hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/ClientInvocation.java

```diff
@@ -124,11 +123,7 @@ public ClientMessage getClientMessage() {
 
     public ClientInvocationFuture invoke() {
         assert (clientMessage != null);
-        try {
-            clientMessage.setCorrelationId(callIdSequence.next());
-        } catch (TimeoutException e) {
-            throw new HazelcastOverloadException("Timed out trying to acquire another call ID.", e);
-        }
+        clientMessage.setCorrelationId(callIdSequence.next());
```

Comment:
We should fix the client to include the invocation. Even though it currently is quite useless because on the client there is no human readable description of the method being called; this needs to be fixed in the future.

Reply:
It will be solved with this issue: https://github.com/hazelcast/hazelcast/issues/11241